cardshowviewtakenpicturesview/src/main/java/br/com/stant/libraries/cardshowviewtakenpicturesview/camera/CameraFragment.java

```diff
@@ -493,13 +574,14 @@ public void onPause() {
     @Override
     public void onStart() {
         super.onStart();
-        mCameraSetup.getFotoapparat().start();
+        Handler handler = new Handler();
+        handler.postDelayed(() -> mFotoapparat.start(), 60);
```

Comment:
why you do this postDelayed ? create a method that the description explain the reason